docs/servo_power_test/index.html

```diff
@@ -17,29 +17,24 @@
     <button id='requestSerialButton'>Start</button>
     <button id='haltButton'>Cancel</button>
     <button id='downloadButton'>Download</button>
-    <button id='analyzeButton'>Analyze displayed range</button>
```

Comment:
Maybe we should do that in the later commit, but the id should be kebab-case (sorry I should have notice that earlier...) https://google.github.io/styleguide/htmlcssguide.html#id_Attributes